### ISSUE TITLE
Latest graphql causes error

### ISSUE THREAD

**Issue by jthegedus** [Canner/gqlify#41]
**Describe the bug**
the auto-installed dependency `graphql 14.1.1` throws the following error:

<details>
<summary>log</summary>

```
firebase-gcp-examples/fb-functions-graphql/gqlify/api on  graphql-examples [?] is 📦 v1.0.0 via ⬢ v8.15.0 took 30s
➜ node src/index.js
Starting Gqlify...

  - Gqlify Models
    - Model User (generated from 'User')
    Field: id `ID!`
    Field: username `String!`
    Field: email `String`
    Field: books `[Book!]!` @relation(name: UserAndBookOnbooks)


    - Model Book (generated from 'Book')
    Field: id `ID!`
    Field: name `String!`
    Field: author `User!` @relation(name: UserAndBookOnbooks)


  - Relations
    Relation UserAndBookOnbooks
      * Type: Bidirectional One-to-Many
      * Relationship: 1-* on `User`-`Book`


/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/graphql/type/schema.js:290
          throw _iteratorError3;
          ^

TypeError: _this.defBuilder._makeInputValues is not a function
    at fields (/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/@gqlify/server/lib/rootNode.js:172:72)
    at resolveThunk (/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/graphql/type/definition.js:374:40)
    at defineInputFieldMap (/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/graphql/type/definition.js:857:18)
    at GraphQLInputObjectType.getFields (/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/graphql/type/definition.js:838:27)
    at typeMapReducer (/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/graphql/type/schema.js:302:61)
    at typeMapReducer (/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/graphql/type/schema.js:243:12)
    at Array.reduce (<anonymous>)
    at typeMapReducer (/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/graphql/type/schema.js:275:38)
    at Array.reduce (<anonymous>)
    at new GraphQLSchema (/home/jthegedus/projects/firebase-gcp-examples/fb-functions-graphql/gqlify/api/node_modules/graphql/type/schema.js:108:28)
```

</details

Explicitly installing `graphql@14.0.2` solves the issue.

**To Reproduce**
attempt to use the [quickstart](https://www.gqlify.com/docs/quick-start#setup-gqlify-with-apollo-server) locally.

**Expected behavior**
It should not throw this schema error.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.10

**Additional context**
It seems the changes in `14.1.0` about schema assertion may be the cause:
https://github.com/graphql/graphql-js/releases/tag/v14.1.0

Potentially related: https://github.com/graphql/graphql-js/issues/1668

**Comment by wwwy3y3** [comment]
hi @jthegedus thanks for the bug report.

It seems that they changed some internal APIs.

I'll see that I can do here.

**Comment by wwwy3y3** [comment]
hi @jthegedus 
Should be fixed at `v1.1.5`.
You can upgrade `graphql` to lastest and use with gqlify now.

**Comment by jthegedus** [comment]
I can confirm this is fixed! Thanks Will
